src/input.tsx

```diff
@@ -33,7 +33,7 @@ interface IProps {
 /**
  * The input field
  */
-export default class Input extends React.PureComponent<IProps, {}> {
+export default class Input extends React.PureComponent<IProps, unknown> {
```

Comment:
what is `unknown`? ts keyword?

Reply:
Yes. https://mariusschulz.com/blog/the-unknown-type-in-typescript

Basically you have to do checks in comparison to any. As we have to provide something but don’t use it an don’t care, this is ideal. ESLint started to complain about usage of `{}` and suggested `unknown`.